e2e/cypress/integration/multi_team_and_dm/existing_channel_name_spec.js

```diff
@@ -106,7 +108,7 @@ describe('Channel', () => {
 
     it('Mult14635 Should not create new channel with existing private channel name', () => {
         // # Create a new private channel
-        createNewChannel(`unique-private-${Date.now()}`, true).as('channel');
+        createNewChannel('unique-private', true, testTeamId).as('channel');
```

Comment:
i guess this doesnt need the date appended anymore cause the team is guaranteed to be a new team right?

Reply:
That's right. Although not fully guaranteed (since we're just getting parts of uuid), we're appending [unique suffix](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api_commands.js#L86) in the `cy.apiCreateChannel` itself.